lib/common.go

```diff
@@ -89,6 +89,15 @@ func (a *ACBuild) lock() error {
 
 	a.lockFile, err = os.OpenFile(a.LockPath, os.O_CREATE|os.O_RDWR, 0644)
 	if err != nil {
+		switch err1 := err.(type) {
+		case *os.PathError:
+			switch err2 := err1.Err.(type) {
+			case syscall.Errno:
+				if err2 == syscall.EACCES {
+					fmt.Fprintf(os.Stderr, "you'll need to run this as a user with appropriate privileges\n")
```

Comment:
errwrap isn't currently used in acbuild, maybe it should be. The changes to incorporate it would be outside the scope of this PR I feel.

How's that look? Instead of printing to stderr I just generate a different error to return.